app/lib/ui/components/component_detail_screen.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:ods_flutter/guidelines/spacings.dart';
+import 'package:ods_flutter_demo/ui/components/component.dart';
+import 'package:ods_flutter_demo/ui/utilities/display_image.dart';
+
+class ComponentDetailScreen extends StatelessWidget {
+  final OdsComponent component;
+
+  const ComponentDetailScreen({required this.component});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text(component.title),
+        ),
+        body: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            displayImage(
+              component.imageResourceName,
+              MediaQuery.of(context).size.width,
+            ),
+            Padding(
+              padding: const EdgeInsets.all(spacingM),
+              child: Text(
+                component.description,
+                style: TextStyle(fontSize: spacingM, letterSpacing: 0.5),
+              ),
+            ),
+            Expanded(
+                child: ListView.builder(
+              itemCount: component.variants.length,
+              itemBuilder: (BuildContext context, int index) {
+                return VariantEntry(variant: component.variants[index]);
+              },
+            ))
+          ],
+        ));
+  }
+}
+
+class VariantEntry extends StatelessWidget {
+  final Variant variant;
+
+  const VariantEntry({required this.variant});
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+      leading: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Icon(Icons.play_circle_outline),
```

Comment:
This icon should be the same as in the Android app. Here it is a grey icon, on Android it is black.